circuit/Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2021"
 plotters = { version = "0.3.0" }
 rand = "0.8"
 hex = { version = "0.4", default-features = false, features = ["alloc"] }
-halo2 = { package = "halo2_proofs", git = "https://github.com/privacy-scaling-explorations/halo2", tag = "v2023_02_02" }
+halo2 = { package = "halo2_proofs", git = "https://github.com/privacy-scaling-explorations/halo2", tag = "v2023_02_02", features = ["dev-graph"] }
```

Comment:
This feature should be enabled only when we are testing. So probably add [dev-dependencies] and list copy the halo2 dep with this feature enabled

Reply:
I tried to just copy the dependencies. 
But, it does not work since the feature should be enabled.

So, I just removed the `dev-graph` feature since it is for development.
https://github.com/eigen-trust/protocol/pull/169/commits/345abec5864f526141995961b9c92d770846d4c5

Reply:
Ok, it can be removed if its not used in tests anymore. If you ever need it again in the future, we should have two version:
```toml
[dependencies]
halo2 = { package = "halo2_proofs", git = "https://github.com/privacy-scaling-explorations/halo2", tag = "v2023_02_02" }
// ...
[dev-dependencies]
halo2 = { package = "halo2_proofs", git = "https://github.com/privacy-scaling-explorations/halo2", tag = "v2023_02_02", features = ["dev-graph"] }
```

But for now its ok to exclude it